### PR TITLE
Add Federico Bongiovanni BIO for Steering election

### PR DESCRIPTION
**Pull request by fedebongio**
Adding my BIO PR

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by jdumars** [comment]
/approve
/hold
unhold after election officers review

**Review by idvoretskyi — APPROVED** [review]
/lgtm

File: events/elections/2020/README.md [hunk 1]
```diff
@@ -161,6 +161,8 @@ the [Steering Committee Election Charter]
 |                    Name                    | Organization/Company |                        GitHub                        |
 |:------------------------------------------:|:--------------------:|:----------------------------------------------------:|
 | [Jane Containerface](./biotemplate.md)     |      ExampleCo       | [@github](https://github.com)                        |
```
Comment (tpepper) [inline events/elections/2020/README.md:163]: Should remove this line, but that will fall out as the set of these are merged/rebased.

/lgtm

**Comment by tpepper** [comment]
One additional nit:  Can you please trim this down just a tad?  You're maybe 10% over the 300 word limit, even if excluding  markdown/template/urls which inflate things some.

**Comment by fedebongio** [comment]
@tpepper shrinked to ˜290 words, let me know if it looks better! and thank you

**Comment by jdumars** [comment]
@fedebongio if you can fix the MC we'll get this approved

**Comment by tpepper** [comment]
/lgtm
/unhold
Thanks for updating!

**Pull request merged**